views/clickhouse.go

```diff
@@ -75,13 +86,38 @@ func (c *ClickhouseClient) QueryRealtimeViewsEvents(ctx context.Context, spec Qu
 }
 
 func buildRealtimeViewsEventsQuery(spec QuerySpec) (string, []interface{}, error) {
-	var query squirrel.SelectBuilder
-	if spec.From == nil && spec.To == nil {
-		query = currentEventsQuery(spec)
-	} else {
-		query = timeRangeEventsQuery(spec)
+	query := squirrel.Select(
+		"count(distinct session_id) as view_count").
+		From("viewership_events").
+		Where("user_id = ?", spec.Filter.UserID).
+		Where("server_timestamp > (toUnixTimestamp(now() - 30)) * 1000").
+		Limit(maxClickhouseResultRows + 1)
+	return toSqlWithFiltersAndBreakdown(query, spec)
+}
+
+func buildTimeSeriesRealtimeViewsEventsQuery(spec QuerySpec) (string, []interface{}, error) {
+	query := squirrel.Select(
+		"timestamp_ts",
+		"count(distinct session_id) as view_count",
+		"sum(buffer_ms) / (sum(playtime_ms) + sum(buffer_ms)) as buffer_ratio",
+		"sum(if(errors > 0, 1, 0)) / count(*) as error_rate").
```

Comment:
Won't `sum(if(errors > 0, 1, 0))` count all rows tho, not deduped by `session_id`?

So if we have 2 rows with errors for the same `session_id`, this would become `2 / 1 = 200%` error rate.

Not sure what's the right approach tho, maybe updating the error sum not to count duplicate `session_id`s, but I'm not sure how to do that.

Reply:
I think that `viewership_sessions_by_minute` already includes only distinct sessions, so "in theory", it should be the same. But... "in practice", we sometimes experienced duplicated entries for the last minute, so I guess we should not change it `distinct_session_id`. @ecmulli wdyt?

Reply:
Updated to `distinct` as suggested by Evan. However, if you think we should change it, then I'll fire a separate PR.